src/constants.ts

```diff
@@ -0,0 +1,23 @@
+export const PACKAGE_NAME = "@monogram/eslint-config"
+
+export const ESLINT_FILENAME = '.eslintrc.js' 
+
+export const availableConfigs = ["next", "node"] as const
```

Comment:
let's follow the constant case for the constants here
```suggestion
export const AVAILABLE_CONFIGS = ["next", "node"] as const
```